lua/gatekeeper_config.lua

```diff
@@ -20,15 +20,19 @@ function gatekeeper_init()
 	local llsf = require("lls")
 	local lls_conf = llsf(net_conf, numa_table)
 
+	local gk_conf
+	local gt_conf
+
 	if gatekeeper_server == true then
 		local gkf = require("gk")
-		local gk_conf, ggu_lcore = gkf(net_conf, numa_table)
+		local gk_config, ggu_lcore = gkf(net_conf, numa_table)
```

Comment:
Just declare "local ggu_lcore` before this line and drop the "local" at the beginning of this line to avoid the copy being none on the line below.